### PR TITLE
Bump plugins to their latest versions

### PR DESCRIPTION
**Pull request by rji**

### CODE REVIEW

**Comment by ssk2** [comment]
🚀

**Comment by rji** [comment]
It looks like we might have a gap in our CI reporting the build status here. The `docker build` actually failed (via this notification I got in Slack): https://jenkins.mesosphere.com/service/jenkins/job/Jenkins/job/jenkins-dcos-publish-docker_dev/177/

**Comment by rji** [comment]
Ping @mesosphere/velocity-team — does anyone have a chance to look at this in the upcoming sprint? Unfortunately I don't think I'll have spare cycles to determine why CI failed and fix it. Otherwise I can abandon this PR in favor of some future work / plugin version bump.

File: pom.xml [hunk 1]
```diff
@@ -32,53 +32,53 @@
 		<blueocean-rest-impl.version>1.0-alpha-4</blueocean-rest-impl.version>
 		<blueocean-web.version>1.0-alpha-4</blueocean-web.version>
 		<!-- Pipeline (nee Workflow) plugins -->
-		<durable-task.version>1.11</durable-task.version>
+		<durable-task.version>1.12</durable-task.version>
 		<pipeline-build-step.version>2.2</pipeline-build-step.version>
-		<pipeline-input-step.version>2.0</pipeline-input-step.version>
-		<pipeline-rest-api.version>1.6</pipeline-rest-api.version>
+		<pipeline-input-step.version>2.1</pipeline-input-step.version>
+		<pipeline-rest-api.version>1.67</pipeline-rest-api.version>
```
Comment (colin-msphere) [inline pom.xml]: This is not a valid version for this plugin. The available versions are `1.7` or `2.0` according to https://updates.jenkins-ci.org/download/plugins/pipeline-rest-api/

Comment (colin-msphere) [inline pom.xml]: In the Update Center, this plugin points to `stage-view`. I am not sure why.

Comment (rji) [inline pom.xml]: Thanks @colin-msphere, looks like I didn't delete the 6!

**Pull request merged**